IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CommunicationsApplianceTypeOpticalLineTerminal/DocPropertySet.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_CommunicationsApplianceTypeOpticalLineTerminal" UniqueId="5d11e497-ab53-a92b-61a8-8a54a11535b3" ApplicableType="IfcCommunicationsAppliance/USERDEFINED" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
This is a usage case. Applicability based on `USERDEFINED` is definitely problematic.